src/components/Blinds/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {useState} from "react";
+import "./style.css";
+
+import blindsOpened from "./images/blinds-open.svg";
+import blindsClosed from "./images/blinds-closed.svg";
+
+const Blinds = ({state}) => {
```

Comment:
to už je taková drobnost ale propsy které se používají na nastavení nějakého počátečního stavu se většinou nazývají jako "initialState" aby sis to nespletla s proměnnou která něco mění přímo